homeassistant/components/media_player/group.py

```diff
@@ -0,0 +1,15 @@
+"""Describe group states."""
+
+
+from homeassistant.components.group import GroupIntegrationRegistry
+from homeassistant.const import STATE_OK, STATE_PROBLEM
+from homeassistant.core import callback
+from homeassistant.helpers.typing import HomeAssistantType
+
+
+@callback
+def async_describe_on_off_states(
+    hass: HomeAssistantType, registry: GroupIntegrationRegistry
+) -> None:
+    """Describe group on off states."""
+    registry.on_off_states([STATE_PROBLEM], STATE_OK)
```

Comment:
hmm this seems like weird states for the media player?

Reply:
Its also kinda weird that a `binary_sensor` with `DEVICE_CLASS_LOCK`'s `on` state is `unlocked` but a `lock` unlocked state is `off`

```# On means open (unlocked), Off means closed (locked)
DEVICE_CLASS_LOCK = "lock"```

Maybe I should switch `lock` but that seems like a breaking change.

Reply:
```
group/__init__.py:    STATE_PROBLEM,
group/__init__.py:    (STATE_PROBLEM, STATE_OK),
homekit_controller/media_player.py:    STATE_PROBLEM,
homekit_controller/media_player.py:            return STATE_PROBLEM
```

Hmm.. STATE_PROBLEM really doesn't appear to be used anymore or I'm misunderstood the history here.  I'm going to adjust `media_player` to be more sane

Reply:
yeah that's also weird

Reply:
Ideally an open cover, an unlocked binary sensor and an unlocked lock are all treated as the same.

Reply:
But then we get to do a different grouping based on a device class ? Not sure if we should go down that rabbit hole because it means that grouping 2 binary senors (one with lock device class) doesn't work as expected.

Reply:
Swapping lock seems ok actually since we didn't support grouping the state with other domains before

Reply:
Added a test, we can't switch it as it would be a breaking change.